### ISSUE TITLE
Authorization with Twitter, not collecting email

### ISSUE THREAD

**Issue by shtefcs** [RocketChat/Rocket.Chat#2039]
When user register via Twitter, we don't collect email.

Probably cause twitter not giving email.

But do you think that is good ? 

Most important thing is to collect user email so you can send him updates, contact via email.

Do you think, it should ask for email too after successful twitter authorization ?

Peace

**Comment by engelgabriel** [comment]
Indeed, twitter doesn't give the email. Its up to you if you think you should also ask for an email and if you want to ask for validation.

**Comment by shtefcs** [comment]
Gabi, is there easy way to setup that from admin, or its need customization ?
